### DISCUSSION TITLE
Question about extra_label_selector

### DISCUSSION THREAD

**Discussion by lkananowicz** [vectordotdev/vector#17626] [Q&A] [answered]
Hi, 

not sure if this was already answered. I am new to Vector configuration and I wonder what is the correct way to specify multiple labels in extra_label_selector. For example: 

`extra_label_selector = "app=production,app=staging,app=scheduler-production,app=other-app-production,app=other-app-staging"` does not seem to work, I guess because of duplicated key `app`?

Thanks,
Leon

**Comment by spencergilbert** [comment, marked as answer]
I believe you should be able to use [set based](https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#set-based-requirement) selection, so something like `app in(production, staging, ...)`.

Can you give that a try?

**Comment by lkananowicz** [comment]
Thanks, it is working now :+1:
